tests/Microsoft.Identity.Test.Unit.net45/CoreTests/Telemetry/TelemetryHelperTests.cs

```diff
@@ -14,7 +14,7 @@ public class TelemetryHelperTests
     {
         private const string CorrelationId = "thetelemetrycorrelationid";
         private const string ClientId = "theclientid";
-        private _TestEvent _trackingEvent;
+        internal _TestEvent _trackingEvent;
```

Comment:
Is this being used somewhere outside of the class? If not, does it need to be internal?